### DISCUSSION TITLE
where is docker self host files ? link to downlaod

### DISCUSSION THREAD

**Discussion by zakidze** [tolgee/tolgee-platform#2259] [Q&A] [answered]
hello where is docker self host files where to downlaod

**Comment by JanCizmar** [comment, marked as answer]
Hello,

You can install Tolgee via DockerHub. https://hub.docker.com/r/tolgee/tolgee

You can read more about running with docker in the docs: https://tolgee.io/platform/self_hosting/running_with_docker

**Reply by zakidze** [reply]
thank you soo much
